examples/guided.py

```diff
@@ -365,31 +494,44 @@ def perform_installation(mountpoint):
 			for superuser, user_info in archinstall.arguments.get('superusers', {}).items():
 				installation.user_create(superuser, user_info["!password"], sudo=True)
 
-			if (timezone := archinstall.arguments.get('timezone', None)):
+			if (timezone := archinstall.arguments.get('timezone', None)) :
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.